command/agent/command.go

```diff
@@ -70,6 +70,18 @@ func (c *Command) readConfig() *Config {
 	// Server-only options
 	flags.IntVar(&cmdConfig.Server.BootstrapExpect, "bootstrap-expect", 0, "")
 
+	// Client-only options
+	flags.StringVar(&cmdConfig.Client.StateDir, "state-dir", "", "")
+	flags.StringVar(&cmdConfig.Client.AllocDir, "alloc-dir", "", "")
+	flags.StringVar(&cmdConfig.Client.NodeID, "node-id", "", "")
+	flags.StringVar(&cmdConfig.Client.NodeClass, "node-class", "", "")
+
+	var servers string
+	flags.StringVar(&servers, "servers", "", "")
+
+	var meta []string
```

Comment:
So far we have been putting all of the type declarations at the top so that we don't have to break up these blocks.